src/server_manager/ui_components/outline-server-settings.html

```diff
@@ -247,71 +209,11 @@ <h3>[[localize('settings-metrics-header')]]</h3>
           this.fire(metricsSignal);
         }
       },
-      setKeysPortEditing: function() {
-        if (!this.isAccessKeyPortEditable) { 
-          return;
-        }
-        this.showKeysPortButtons = true;
-        const invalid = this.$.keysPortInput.invalid;
-        this.$.keysPortSaveButton.disabled = invalid;
-        this.$.keysPortCancelButton.disabled = false;
-      },
-      setKeysPortSaved: function() {
-        this.serverPortForNewAccessKeys = Number(this.$.keysPortInput.value);
-        this.showKeysPortButtons = false;
-        this.$.keysPortInput.readonly = false;
-      },
-      setKeysPortErrorState: function(message) {
-        const input = this.$.keysPortInput;
-        this.enableKeysPortButtons = true; 
-        input.invalid = true;
-        input.errorMessage = message;
-        input.readonly = false;
-        input.focus();
-      },
-      saveKeysPort: function() {
-        const input = this.$.keysPortInput;
-        if (Number(input.value) === this.serverPortForNewAccessKeys) {
-          this.cancelKeysPortEdit();
-          return;
-        };
-        this.enableKeysPortButtons = false;
-        input.readonly = true;
-        input.invalid = false;
-        this.fire('ChangePortForNewAccessKeysRequested', {
-          port: Number(this.$.keysPortInput.value), 
-          serverSettings: this
-        });
-      },
-      cancelKeysPortEdit: function() {
-        this.$.keysPortInput.value = this.serverPortForNewAccessKeys;
-        this.$.keysPortInput.invalid = false;
-        this.showKeysPortButtons = false;
-      },
-      _handleKeysPortInputKeyUp: function(e) {
-        let input = this.$.keysPortInput;
-        const badInputMessage = this.localize('error-keys-port-bad-input');
-        if (e.key === 'Escape') {
-          this.cancelKeysPortEdit();
-          input.blur();
-          return; 
-        } else if (e.key === 'Enter') {
-          if (!input.invalid) {  
-            this.saveKeysPort();
-            input.blur();
-          }
-          return;
-        }
-        let port = Number(input.value);
-        if (Number.isNaN(port) || (port < 1 || port > 65535) || !Number.isInteger(port)) {
-          input.invalid = true;
-          this.$.keysPortSaveButton.disabled = true;
-          input.errorMessage = badInputMessage;
-        } else {
-          input.invalid = false;
-          this.$.keysPortSaveButton.disabled = false;
-        }
-      }
+      _keysPortValidator: function(value) {
```

Comment:
nit: use a verb for methods, i.e. `_validateKeysPort`

Reply:
Done